packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -270,8 +300,8 @@ contract ChugSplashManager is Owned {
         ChugSplashBundleState storage bundle = bundles[activeBundleId];
 
         require(
-            bundle.status != ChugSplashBundleStatus.COMPLETED,
-            "ChugSplashManager: bundle has already been completed"
+            bundle.status == ChugSplashBundleStatus.APPROVED,
```

Comment:
Changed this from `!= ChugSplashBundleStatus.COMPLETED` to `== ChugSplashBundleStatus.APPROVED` because there's now a `CANCELLED` attribute on `ChugSplashBundleStatus`.